src/contracts/BCoWFactory.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.25;
+
+import {BCoWPool} from './BCoWPool.sol';
+
+import {BFactory} from './BFactory.sol';
+import {IBFactory} from 'interfaces/IBFactory.sol';
```

Comment:
unused

Reply:
it's used in the `@inheritdoc` for `newBPool`, most likely a bug with the linter

Reply:
should we ignore it?